private/README.md

```diff
@@ -0,0 +1,5 @@
+﻿# Private Functions
+
+Contains private function files, for use within the module only.
+
+We name private functions with `VerbNoun` (no joining dash). This helps quickly identify private functions when used within the module.
```

Comment:
Should we start sentences like this with: "By convention, we" ...

Reply:
Done